packages/wallet/test/SingleAddressWallet/methods.test.ts

```diff
@@ -1,16 +1,15 @@
 /* eslint-disable max-len */
 import * as mocks from '../mocks';
+import { AddressType, GroupedAddress } from '@cardano-sdk/key-management';
 import { AssetId, createStubStakePoolProvider } from '@cardano-sdk/util-dev';
 import { Cardano } from '@cardano-sdk/core';
-import { InitializeTxProps, KeyManagement, SingleAddressWallet, setupWallet } from '../../src';
+import { InitializeTxProps, SingleAddressWallet, setupWallet } from '../../src';
 import { firstValueFrom, skip } from 'rxjs';
+import { getPassword, testAsyncKeyAgent } from '../../../key-management/test/mocks';
 import { dummyLogger as logger } from 'ts-log';
 import { mockChainHistoryProvider, mockRewardsProvider, utxo } from '../mocks';
 import { waitForWalletStateSettle } from '../util';
 
-jest.mock('../../src/KeyManagement/cip8/cip30signData');
```

Comment:
Jest, with our existing config, wasn't happy referring to a sibling package, so I just substituted the assertion to inspect the return value. This is easier than managing tedious path references, although I'd be happy to change it back if you can see why it's failing

Reply:
I think this is fine